lexers/embedded/hare.xml

```diff
@@ -0,0 +1,101 @@
+<lexer>
+  <config>
+    <name>Hare</name>
+    <alias>hare</alias>
+    <filename>*.ha</filename>
+    <mime_type>text/x-hare</mime_type>
+  </config>
+  <rules>
+    <state name="whitespace">
+      <rule pattern="[\s\n]+">
+        <token type="TextWhitespace"/>
+      </rule>
+      <rule pattern="@[a-z]+">
+        <token type="NameDecorator"/>
+      </rule>
+      <rule pattern="//.*\n">
+        <token type="CommentSingle"/>
+      </rule>
+    </state>
```

Comment:
I don't see a good reason to have separate `whitespace` and `statements` states, maybe they should just be folded into the `root` state?